modules/cert-manager-override-arguments.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-override-arguments_{context}"]
+= Customizing cert-manager by overriding arguments from the cert-manager Operator API
+
+You can override the supported arguments for the {cert-manager-operator} by adding a `spec.controllerConfig` section in the `CertManager` resource.
```

Comment:
@xingxingxia Can we say - "You can override the **supported** arguments...."? Since we cannot override existing i.e default arguments, I thought of using "supported arguments"

Reply:
OK